interpretation/ww_head.py

```diff
@@ -699,8 +703,10 @@ def process(message, isGameMaster=False, isAdmin=False, isPeasant=False):
                 # TODO
                 return todo()
             if is_command(message, ['heal', 'life', 'save'], True) and user_role == "Witch":
-                # TODO
-                return todo()
+                msg = "**Usage:** Use potion of life.\n\n`" + prefix + "save <player>`\n\n"
+                msg += "**Example:** `" + prefix + "save @Randium#6521`\nThe command is compatible with emojis as a replacement for user mentions. "
+                msg += "This command can only be used by the Witch."
```

Comment:
Same for this one. Your potion will save EVERYBODY that is getting attacked during the night. No mentioning players.

Reply:
LMAO